samples/analyze-streaming-automl-object-tracking.js

```diff
@@ -0,0 +1,98 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+async function main(
+  path = 'YOUR_LOCAL_FILE',
+  projectId = 'YOUR_GCP_PROJECT',
+  modelId = 'YOUR_AUTOML_MODELID'
+) {
+  // [START video_streaming_automl_object_tracking_beta]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // const path = 'Local file to analyze, e.g. ./my-file.mp4';
+  // const modelId = 'autoMl model'
+  // const projectId = 'Your GCP Project'
+
+  const {StreamingVideoIntelligenceServiceClient} =
+    require('@google-cloud/video-intelligence').v1p3beta1;
+  const fs = require('fs');
+
+  // Instantiates a client
+  const client = new StreamingVideoIntelligenceServiceClient();
+
+  // Streaming configuration
+  const modelPath = `projects/${projectId}/locations/us-central1/models/${modelId}`;
```

Comment:
NIT: Rename this variable to `modelName` (which is more precise about what it is). That way, you can shorten line 43 to just:

```
modelName,
```